app/routes.py

```diff
@@ -1,5 +1,39 @@
+from flask import request
 from app import app
 
 @app.route('/')
 def inded():
 	return "Hello, World!"
+
+# "methods = post" will allow for only POST requests
+@app.route('/create-user', methods=['POST'])
+def create_user():
+	emailAddress = None
+	firstName = None
+	lastName = None
+	password = None
+	# processes JSON data and converts to variables
```

Comment:
In general, we don't need to add comments for code that is clear and easily readable to understand what is happening. Comments like this are good in tutorials but in production code we don't need them.